package.json

```diff
@@ -26,14 +26,14 @@
     "android"
   ],
   "dependencies": {
-    "mkpath": ">=1.0.0",
-    "xml2js": ">=0.4",
-    "rimraf": ">=2.4",
-    "node-version-compare": ">=1.0.1",
-    "plist": ">=1.2.0",
-    "glob": "^7.1.3",
-    "xcode": "^2.0.0",
-    "shelljs": "^0.8.1"
+    "mkpath": "^1.0.0",
+    "xml2js": "0.5.0",
```

Comment:
Because of their cavalier approach to version numbering, I have pinned this. The other libraries are either dormant or have adopted more reasonable practices in this regard